upstream-community-operators/gpu-operator/1.0.0/gpu-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,208 @@
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - configmaplists
+                - secrets
+                - serviceaccounts
+                - nodes
+              verbs:
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - list
+                - create
+                - watch
+                - update
+            - apiGroups:
+                - sro.openshift.io
+              resources:
+                - '*'
+              verbs:
+                - '*'
+            - apiGroups:
+                - scheduling.k8s.io
+              resources:
+                - priorityclasses
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+            - apiGroups:
+                - batch
+              resources:
+                - jobs
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+          serviceAccountName: gpu-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - gpu
+    - cuda
+    - compute
+    - operator
+    - 'deep learning'
+    - monitoring
+    - tesla
+  links:
+    - name: 'GPU Operator home'
+      url: 'https://github.com/NVIDIA/gpu-operator'
+    - name: Support
+      url: 'https://github.com/NVIDIA/gpu-operator/issues/new'
+    - name: 'OpenShift on GPU Install Guide'
+      url: 'https://docs.nvidia.com/datacenter/kubernetes/openshift-on-gpu-install-guide/index.html'
+    - name: 'Docker Repository'
+      url: 'https://hub.docker.com/u/nvidia'
+  maintainers:
+    - name: NVIDIA
+      url: 'https://github.com/NVIDIA/gpu-operator/issues/new'
+  provider:
+    name: 'NVIDIA Corporation'
+  replaces: gpu-operator.v0.0.0
```

Comment:
Remove this line, you do not have a gpu-operator.v0.0.0 CSV installed on OperatorHub. 
The next commit can replace the version that is already posted in OperatorHub. 

E.g. if you have an new PR with gpu-operator.v2.0.0 this would have an entry "replaces: gpu-operator.v1.0.0"

Currently you're not replacing anything because this PR is the first entry.

Reply:
doh. seems obvious now. 🤦‍♂ Thanks.